packages/build-utils/test/fixtures/22-pnpm/vercel.json

```
@@ -0,0 +1,5 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@vercel/static-build" }],
+  "probes": [{ "path": "/", "mustContain": "pnpm version: 6.32.1" }]
```

```suggestion
  "probes": [{ "path": "/", "mustContain": "pnpm version: 6" }]
```

Lets test only major since we might be bumping the minor and patch version frequently